src/features/chains/hooks.ts

```diff
@@ -31,22 +28,3 @@ export function useChainDisplayName(chainName: ChainName, shortName = false) {
   const multiProvider = useMultiProvider();
   return getChainDisplayName(multiProvider, chainName, shortName);
 }
-
-/**
- * Returns chainMetadata object with warp core token chains
- */
-export function useTokenChainsMetadata() {
```

Comment:
@Xaroz Note I made a change after you wrote this and now the ChainMetadata map in the store's multiprovider only includes the token chain anyway. So this hooks is no longer needed :)